client/app/components/contact-form.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+import config from '../config/environment';
+const isTest = config.APP.STUB_EXTERNAL_MOCK;
+
+export default Ember.Component.extend({
+
+  districtContactEmail: '',
+  districtContactSubmitted: false,
+  districtContactErrored: false,
```

Comment:
All these properties were prefixed with `districtContact` because this code was previously within the main landing page component, so it was sort of a "namespacing" measure. Since this contact form is a component itself – a much better approach, I might add! – we can probably drop the prefixes. (same with `submitDistrictContact`, could just be `submit`)